tests/builder/test_base.py

```diff
@@ -1,9 +1,25 @@
 import sys
+from pathlib import Path
 from unittest import mock
 
+import pytest
+
 from rpmlb.builder.base import BaseBuilder
 
 
+@pytest.fixture
+def empty_spec_path(tmpdir):
+    """Empty spec file path"""
```

Comment:
Sorry one more thing.
Did you see below below `pushd_tmp_dir` function? It is same kind of function with `empty_spec_path`, isn't it?
Why do you create same kind of method as different file and different implementation?
Is it possible to implement it on DRY (Don't repeat yourself) concept?

Also variable `tmp_dir` is better than `tmpdir`?
https://github.com/sclorg/rpm-list-builder/blob/master/tests/helper.py#L23